docs/completeExamples/completeExamples.xml

```diff
@@ -0,0 +1,485 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<Events xmlns="event-logging:3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="event-logging:3 file://event-logging-v3.0.0.xsd" Version="3.0.0">
+    <Event>
```

Comment:
I think it would be useful to add an xml comment above or below <Event> to give a brief description of what the example is trying to demonstrate.